### PR TITLE
Reformat YAML [13423]

### PR DESCRIPTION
**Pull request by juanlofer-eprosima**
Signed-off-by: Juan López Fernández <juanlopez@eprosima.com>

### CODE REVIEW

File: docs/resources/examples/change_domain.yaml [hunk 1]
```diff
@@ -7,10 +7,10 @@
 # Allowing FastDDS and ROS2 HelloWorld demo examples topics
 
 allowlist:
-  [
-    {name: "HelloWorldTopic", type: "HelloWorld"},                          # 1
-    {name: "rt/chatter", type: "std_msgs::msg::dds_::String_"},             # 2
-  ]
+  - name: "HelloWorldTopic"                                                 # 1
```
Comment (jparisu) [inline docs/resources/examples/change_domain.yaml:10]: In general:
Modify docs examples so they are links to the examples in resources. Aplpy the suggestions to the one file that remains (recommended the one in resources).
(This should have been like that from the beggining, I do not know at what time it changed)

File: resources/configurations/examples/change_domain.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ SimpleParticipant_domain0:                                                  # 3
   domain: 0                                                                 # 5
 
 ##################################
-# SIMPLE PARTICIPANT DOMAIN 1
+# SIMPLE PARTICIPANT DOMAIN 0
```
Comment (jparisu) [inline resources/configurations/examples/change_domain.yaml]: why this change?

File: resources/configurations/examples/change_domain.yaml [hunk 3]
```diff
@@ -32,8 +32,8 @@ SimpleParticipant_domain1:                                                  # 6
 ##################################
 # CONFIGURATION DESCRIPTION
 
-# This configuration example configures a DDS Router to listen to every message published in two different domains
-# and transmit those messages trough the other domain
+# With this example, a DDS Router listens to every message published in two different domains and transmit
```
Comment (jparisu) [inline resources/configurations/examples/change_domain.yaml]: In general:
Be careful, these are **Configuration examples**, not **Router examples**.
The explanation of each example must be done in the documentation and not in these files.

File: resources/configurations/examples/ros_discovery_client.yaml [hunk 4]
```diff
@@ -28,19 +27,12 @@ ClientROS2:                                                       # 5
...
+        - ip: "127.0.0.1"                                         # 12
           port: 11888                                             # 13
-        },                                                        # 14
-      ]
-    },                                                            # 15
-  ]
+                                                                  # 14
 
 ##################################
 # CONFIGURATION DESCRIPTION
```
Comment (jparisu) [inline resources/configurations/examples/ros_discovery_client.yaml:38]: The configuration description is not correct.

File: resources/configurations/examples/ros_discovery_client.yaml [hunk 5]
```diff
@@ -53,9 +45,9 @@ ClientROS2:                                                       # 5
 #
 # The last one will act as a Discovery Client.
 # It will connect to a remote Discovery Server and communicate to every node connected to it, and route data
-# from the other interfaces.
+#  from the other interfaces.
```
Comment (jparisu) [inline resources/configurations/examples/ros_discovery_client.yaml]: I like this format as well, but we do not use it (try to convince @rsanchez15 )
```suggestion
# from the other interfaces.
```

File: resources/configurations/examples/ros_discovery_server.yaml [hunk 6]
```diff
@@ -7,34 +7,29 @@
...
+  type: local-discovery-server                     # 6
+  ros-discovery-server: true                       # 7
+  id: 1                                            # 8
+  listening-addresses:                             # 9
+    - ip: "127.0.0.1"                              # 10
+      port: 11888                                  # 11
+                                                   # 12
 
 ##################################
 # CONFIGURATION DESCRIPTION
```
Comment (jparisu) [inline resources/configurations/examples/ros_discovery_server.yaml:35]: Incorrect description

File: resources/configurations/examples/wan_client.yaml [hunk 7]
```diff
@@ -74,25 +66,23 @@ WANClient:                                                        # 6
 # 8: Set the Discovery Server GuidPrefix to <01.0f.02.00.00.00.00.00.00.00.ca.fe> for WANClient.
 
 # 9: Add the addresses where to reach the remote DDS Routers that will connect to.
+# Add as many connection-addresses as needed.
 
 # 10: Set the Remote Discovery Server GuidPrefix to <01.0f.01.00.00.00.00.00.00.00.ca.fe>.
 
 # 11: Connect to a Discovery Server in IP <1.1.1.1> listening in port 11666.
```
Comment (jparisu) [inline resources/configurations/examples/wan_client.yaml:73]: Remove UDP type and say it is by default, or comment that line as a new point

File: resources/configurations/examples/wan_server.yaml [hunk 8]
```diff
@@ -28,14 +28,9 @@ WANServer:                                                        # 6
...
-      port: 11666,                                                # 11
+    - ip: "1.1.1.1"                                               # 10
+      port: 11666                                                 # 11
       transport: "udp"                                            # 12
-    },                                                            # 13
-  ]
-
 
 ##################################
 # CONFIGURATION DESCRIPTION
```
Comment (jparisu) [inline resources/configurations/examples/wan_server.yaml:36]: Change the phrase `the behavior of each of the routers do not change.` to say `it is the same` or somehow explain that is independetly if you are Client or Server.
Now it looks like it says it does not change, but are not equal.

**Review by jparisu — CHANGES_REQUESTED** [review]
Request changes for `resources` yaml files.

File: docs/resources/examples/change_domain.yaml (hunk 1 above)
Comment (juanlofer-eprosima) [reply docs/resources/examples/change_domain.yaml]: :+1:

File: resources/configurations/examples/change_domain.yaml (hunk 2 above)
Comment (juanlofer-eprosima) [reply resources/configurations/examples/change_domain.yaml]: :+1: Fixed in 145bc84c186a1d673f7ef83ee3308e58c264d503

File: resources/configurations/examples/change_domain.yaml (hunk 3 above)
Comment (juanlofer-eprosima) [reply resources/configurations/examples/change_domain.yaml]: :+1:

File: resources/configurations/examples/ros_discovery_client.yaml (hunk 5 above)
Comment (juanlofer-eprosima) [reply resources/configurations/examples/ros_discovery_client.yaml]: :+1: Changed in 145bc84c186a1d673f7ef83ee3308e58c264d503

File: resources/configurations/examples/ros_discovery_server.yaml (hunk 6 above)
Comment (juanlofer-eprosima) [reply resources/configurations/examples/ros_discovery_server.yaml:35]: :+1:

File: resources/configurations/examples/wan_client.yaml (hunk 7 above)
Comment (juanlofer-eprosima) [reply resources/configurations/examples/wan_client.yaml]: :+1:

File: resources/configurations/examples/wan_server.yaml (hunk 8 above)
Comment (juanlofer-eprosima) [reply resources/configurations/examples/wan_server.yaml:36]: :+1:

File: docs/rst/user_manual/participants/local_discovery_server.rst [hunk 9]
```diff
@@ -57,36 +57,22 @@ Discovery Server GuidPrefix.
...
+      ros-discovery-server: true                  # ROS Discovery Server id => GuidPrefix = 44.53.02.5f.45.50.52.4f.53.49.4d.41
+
+      listening-addresses:                        # Local Discovery Server Listening Addresses
+        - ip: "127.0.0.1"                         # Use UDP by default
+          port: 11600
+        - ip: "127.0.0.1"
+          port: 11601
+          transport: "tcp"                        # Use TCP transport
+
+        connection-addresses:                     # External Discovery Server Listening Addresses
```
Comment (jparisu) [inline docs/rst/user_manual/participants/local_discovery_server.rst:74]: Incorrect tabulation

**Review by jparisu — CHANGES_REQUESTED** [review]
Small change:

Also, remove two YAML files in `resources/configurations` that are no longer used

File: docs/rst/user_manual/participants/local_discovery_server.rst (hunk 9 above)
Comment (juanlofer-eprosima) [reply docs/rst/user_manual/participants/local_discovery_server.rst]: :+1:

**Review by jparisu — APPROVED** [review]
LGTM
